mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -982,19 +989,33 @@ class LogisticRegressionModel private[spark] (
     throw new SparkException("No training summary available for this LogisticRegressionModel")
   }
 
+  @Since("2.2.0")
+  def binarySummary: BinaryLogisticRegressionTrainingSummary = summary match {
```

Comment:
For convenience. Otherwise, users need to call `model.summary.asInstanceOf[BinaryLogisticRegressionTrainingSummary]` in order to access the extra, binary-specific methods.